raspberryPI3/ros2_ws/src/imu_tools/imu_filter_madgwick/src/imu_filter_ros.cpp

```diff
@@ -448,6 +448,7 @@ void ImuFilterMadgwickRos::publishFilteredMsg(
     // create and publish filtered IMU message
     ImuMsg imu_msg = *imu_msg_raw;
 
+    imu_msg.header.frame_id = FRAME_ID;
```

Comment:
Linked to the previous com,
Line 467 there is this code : imu_msg.header.frame_id= imu_frame_; 

So the coded added here is repetitive

Reply:
yes, I removed all of not using code lines